.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
The flake8 config is included in `setup.cfg`, this should be needed.

Reply:
Thanks for pointing this out. I added the new suppression to `setup.cfg`